biojava-structure/src/test/resources/1hhbCMPND+SRC.ent

```diff
@@ -0,0 +1,18 @@
+HEADER    OXYGEN TRANSPORT                        01-APR-75   1HHB              
```

Comment:
Could you place these new resources files in dir `src/test/resources/org/biojava/nbio/structure/io` ?

Reply:
Done